# Stefan Teleman <stefan.teleman@oracle.com>
# Make sure these structs are always aligned on 8.
# Internal patch. Submitted upstram. Status pending.
--- libgomp/config/posix/sem.h	2014-01-02 17:25:54.000000000 -0500
+++ libgomp/config/posix/sem.h	2014-11-19 08:35:29.670306230 -0500
@@ -48,8 +48,8 @@
 
 struct gomp_sem
 {
-  pthread_mutex_t	mutex;
-  pthread_cond_t	cond;
+  pthread_mutex_t	mutex __attribute__((__aligned__(8)));
+  pthread_cond_t	cond __attribute__((__aligned__(8)));
   int			value;
 };
 
